src/client/dfuse/dfuse_main.c

```diff
@@ -557,8 +557,8 @@ main(int argc, char **argv)
 	 */
 	duns_attr.da_flags = DUNS_NO_REVERSE_LOOKUP;
 	rc = duns_resolve_path(dfuse_info->di_mountpoint, &duns_attr);
-	DFUSE_TRA_INFO(dfuse_info, "duns_resolve_path() on mountpoint returned %d %s",
-		       rc, strerror(rc));
+	DFUSE_TRA_INFO(dfuse_info, "duns_resolve_path() on mountpoint failed: %d (%s)", rc,
```

Comment:
This isn't necessarily a failure, right?

Reply:
True, in fact I've just seen this is a INFO message not an ERROR.  I still think the change is valid however, just for consistency.

Reply:
how is it valid that a return value of 0 would log that it failed?

Reply:
Sorry, I didn't look closely enough at the change in light of Daltons comment.  Consistency with use of strerror is good but you're correct I should not have changed this to failed, I've reverted that part of the change.